migration_to_v2_0_tests/tests/migrated_data_test.py

```diff
@@ -55,6 +57,18 @@ def test_complete_status(client, publication):
     assert publ_detail['layman_metadata']['publication_status'] == 'COMPLETE', f'rest_publication_detail={publ_detail}'
 
 
+@pytest.mark.usefixtures("import_publication_uuids_fixture")
+@pytest.mark.parametrize("layer", LAYERS_TO_MIGRATE, ids=ids_fn)
+def test_layer_thumbnails(client, layer):
+    assert layer.exp_thumbnail_path
+    img_path = f"tmp/artifacts/migration_to_v2_0_tests/layer_thumbnails/{layer.name}_v2_0.png"
+    client.get_layer_thumbnail_by_wms(layer.workspace, layer.name, actor_name=layer.owner, output_path=img_path)
+
+    exp_img_path = layer.exp_thumbnail_path
+    diff_pixels = compare_images(img_path, exp_img_path)
+    assert diff_pixels <= 10, f"diff_pixels={diff_pixels}\nimg_path={img_path}\nexp_img_path={exp_img_path}"
```

Comment:
I would rather have the `10` in some global constant.

Reply:
Moved to constant